dotnet/src/AutoGen.SemanticKernel/SemanticKernelChatCompletionAgent.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// SemanticKernelChatCompletionAgent.cs
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace AutoGen.SemanticKernel;
+
+public class SemanticKernelChatCompletionAgent : IAgent
+{
+    public string Name { get; }
+    private readonly ChatCompletionAgent _chatCompletionAgent;
+    private readonly string _systemMessage;
+
+    public SemanticKernelChatCompletionAgent(ChatCompletionAgent chatCompletionAgent,
+        string systemMessage = "You are a helpful AI assistant")
+    {
+        this.Name = chatCompletionAgent.Name ?? string.Empty;
+        this._chatCompletionAgent = chatCompletionAgent;
+        this._systemMessage = systemMessage;
```

Comment:
`ChatCompletionAgent` processes system message by itself in its `Instructions`, so we don't need to process system message in `SemanticKernelChatCompletionAgent`